app/Transformers/UserTransformer.php

```diff
@@ -30,6 +30,7 @@ public function transform(User $user)
         $profileCustomization = $this->userProfileCustomization($user);
 
         return array_merge($result, [
+            'comments_count' => $user->comments->count(),
```

Comment:
this should be `$user->comments()->count()` otherwise it's just fetching all the comments